components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -473,6 +476,22 @@ export namespace GitpodServer {
         type: GitpodTokenType;
         scopes?: string[];
     }
+    export interface OnboardingState {
+        /**
+         * Whether this Gitpod instance is already confingured with SSO.
+         */
+        readonly isCompleted: boolean;
+
+        /**
+         * Whether any org is .
+         */
+        readonly hasAnyOrg: boolean;
+
+        /**
+         * Whether this
+         */
+        readonly hasAnyOrgWithInactiveSSO: boolean;
+    }
```

Comment:
While reviewing this, I considered we could return the actual counts instead of the booleans, but that also presents challenges for SaaS, so I think the above opaque is a good balance.